pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -516,6 +507,16 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 			newMounterErr)
 	}
 
+	// Check existence of mount point for filesystem volume or symbolic link for block volume
```

Comment:
Can you update comment.

Reply:
This comment describes how `CheckVolumeExistenceOperation` works internally to check volume existence. It's correct.

Reply:
I think @saad-ali means to add a comment about the change? Instead of using volume.volumePath, we need to use volumeMounter.GetPath() because the mount point setup is different for different plugins. We have to be careful about this when adding/modifying code.

Reply:
oh, sorry, because it's already merged, we can add it later.